src/pairings.rs

```diff
@@ -184,6 +187,7 @@ impl_add_binop_specify_output!(MillerLoopResult, MillerLoopResult, MillerLoopRes
 /// Typically, $\mathbb{G}_T$ is written multiplicatively but we will write it additively to
 /// keep code and abstractions consistent.
 #[cfg_attr(docsrs, doc(cfg(feature = "pairings")))]
+#[cfg_attr(feature = "zeroize", derive(Zeroize))]
 #[derive(Copy, Clone, Debug, Default)]
```

Comment:
Oh no, I just noticed a mistake: `Gt::default()` should return `Gt::identity()` which is `Gt(Fp12::one())`, but this derived impl will return `Gt(Fp12::default())` which is `Gt(Fp12::zero())`. Opened https://github.com/zkcrypto/bls12_381/issues/72.